lib/decidim/kids/version.rb

```diff
@@ -4,7 +4,6 @@ module Decidim
   # This holds the decidim-meetings version.
   module Kids
     VERSION = "0.1.0"
-    DECIDIM_VERSION = { git: "https://github.com/decidim/decidim", branch: "develop" }.freeze
```

Comment:
I'd keep this with "0.27.2" so we handle all version related things in the same place